data/sql_updates/create_candidate_detail_view.sql

```diff
@@ -50,12 +51,16 @@ from dimcand
         select distinct on (cand_sk) * from dimcandproperties
             order by cand_sk, candproperties_sk desc
     ) cand_p_most_recent using(cand_sk, cand_id)
-    left join dimcandoffice co on co.cand_sk = dimcand.cand_sk
+    left join dimcandoffice co using (cand_sk)
     left join dimoffice using (office_sk)
     inner join dimparty using (party_sk)
+    left join dimcandproperties dcp using (cand_sk)
+    where dcp.election_yr >= :START_YEAR
```

Comment:
Since this is configurable, we could put a comment in the config to always start with the actual minimum year (not even-numbered cycle) you want, to keep the SQL simple.